libcxx/test/benchmarks/algorithms/modifying/copy.bench.cpp

```diff
@@ -76,6 +90,30 @@ int main(int argc, char** argv) {
 #endif
   }
 
+  // {std,ranges}::copy(random_access_iterator, random_access_iterator, vector<bool>)
```

Comment:
I don't think this benchmark is necessary, since the benchmark above already covers it (`std::vector<int>` is random access).